lib/virtual-net/Cargo.toml

```diff
@@ -14,6 +14,8 @@ base64.workspace = true
 hyper = { workspace = true, optional = true }
 rkyv = { workspace = true, optional = true }
 
+ipnet = "2.10.1"
```

Comment:
Do we really need two new dependencies?
We already have enough of them...

Reply:
I prefer not to implement IP-range logic (such as `contains`) myself if I can use a crate for it.